docs/advanced/settings.md

```diff
@@ -822,6 +839,30 @@ https://your.upgrade.checker-url/v99/checkupgrade
 
   :::
 
+- `nodeUpgradeOption`: Definition of how Harvester must perform node upgrades.
+
+    The node upgrade is an atomic operation, which includes upgrading of the node's RKE2 components and operating system. The upgrade is either fully completed or failed, with no half-finished state.
+
+    To prepare a target node for upgrade, Harvester first attempts to live-migrate all running virtual machines to other nodes. Virtual machines that cannot be live-migrated are automatically shut down to avoid potential disruption and issues during the subsequent upgrade steps.
+
+- `strategy`: Node upgrade strategy.
+
+- `mode`: Mode of node upgrade strategy.
+
+    - `auto`: Node upgrades start automatically. This is the default value.
+
+    - `manual`: Node upgrades are paused until you take specific actions to resume the process.
+
+- `pauseNodes`: List of nodes that must be excluded from automatic upgrades.
+
+    If the `mode` field is set to `manual` and you do not specify any node names in this field, upgrades are paused for all nodes. If the `mode` field is set to `auto`, node names specified in this field are ignored and node upgrades start automatically.
+
```

Comment:
All configs here use the same indention in the doc.
Would be better if `strategy`, `mode` and `pauseNodes` have one more indention under nodeUpgradeOption.

<img width="1008" height="624" alt="Image" src="https://github.com/user-attachments/assets/902efdd2-9682-4f6e-9ab2-7d39485c6bee" />

Note: `imagePreloadOption` also has the same issue